src/web/extractors.rs

```diff
@@ -289,7 +302,20 @@ impl FromRequest for BsoBody {
                 ));
             }
         }
-        let state = req.app_data::<ServerState>().unwrap();
+        let state = match req.app_data::<ServerState>() {
+            Some(s) => s,
+            None => {
+                dbg!("⚠️ Could not load the app state");
+                return Box::new(future::err(
+                    ValidationErrorKind::FromDetails(
```

Comment:
nit: these should probably be ApiErrorKind::Internal but we don't have a `From<ApiErrorKind> for actix_web::Error` (however ValidationErrorKind does).

Our error hierarchy and these from conversions are a bit involved so I'm fine ok w/ validation errors for now